app/controllers/debt_management_controller.rb

```diff
@@ -7,9 +7,10 @@ class DebtManagementController < ApplicationController
 
   def company_list
     return @company_list if @company_list
-    @company_list = Core::ArticleReader.new(COMPANY_LIST_ARTICLE_ID).call do
-      not_found
-    end
+    @company_list = Mas::Cms::Article.find(
+      COMPANY_LIST_ARTICLE_ID,
+      locale: I18n.locale
+    )
     @company_list = ContentItemDecorator.decorate(@company_list)
   end
```

Comment:
I think there is too much going on in this method. Perhaps, the decorating of the company list could be moved into a separate method, wdyt?

Reply:
Humm, seeing for the second time it makes sense move the cms find to a private method. Wdyt:

```
  @company_list = ContentItemDecorator.decorate(resource)

   private
   def resource
     Mas::Cms::Article.find(
       COMPANY_LIST_ARTICLE_ID,
       locale: I18n.locale
     )
   end
```

Reply:
Yes, that is more readable, but in the show method, right?

Reply:
♻️